packages/vite/src/node/plugins/index.ts

```diff
@@ -58,7 +58,7 @@ export async function resolvePlugins(
     depsOptimizerEnabled ? optimizedDepsPlugin(config) : null,
     isBuild ? metadataPlugin() : null,
     !isWorker ? watchPackageDataPlugin(config.packageCache) : null,
-    preAliasPlugin(config),
+    !isBuild ? preAliasPlugin(config) : null,
```

Comment:
https://github.com/rolldown/vite/blob/2ee0e2fab38fbd0dc9660d3406b31da875471d39/packages/vite/src/node/plugins/preAlias.ts#L33-L33
in build mode, depsOptimizer is always false, thus 
https://github.com/rolldown/vite/blob/2ee0e2fab38fbd0dc9660d3406b31da875471d39/packages/vite/src/node/plugins/preAlias.ts#L34-L41
is always false , the plugin do nothing when `build` mode, put the condition outer could reduce a plugin in `build` mode, this could improve performance when integrate `rolldown`